ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/Logger.kt

```diff
@@ -0,0 +1,20 @@
+package com.revenuecat.purchases.ui.revenuecatui
```

Comment:
Can we move this somewhere else other than the root to help organize things? maybe `helpers`?

Reply:
Yup I will change that in a followup PR organizing all these files into packages.